Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -470,19 +470,24 @@ bool MediaPlayerPrivateGStreamer::paused() const
         return false;
     }
 
+#if !defined(GST_DISABLE_GST_DEBUG) || !defined(NDEBUG) || defined(MEDIA_STREAM)
```

Comment:
the mediastream check is incorrect (should be ENABLE_MEDIA_STREAM and check its value) but anyway... the check should runtime, using `isMediaStreamPlayer()` as mentioned earlier.

Reply:
Do you mean I should remove the compile time checks and just keep the runtime one?

Reply:
Move the runtime check before this ifdef is what i mean :)

Reply:
Check last version, IMHO, this is the way to go!